coalib/results/Diff.py

```diff
@@ -352,9 +353,12 @@ def delete_line(self, line_nr):
         """
         Mark the given line nr as deleted. The first line is line number 1.
         """
-        linediff = self._get_change(line_nr)
-        linediff.delete = True
-        self._changes[line_nr] = linediff
+        if (line_nr <= self._length):
```

Comment:
Sorry forgot to mark it WIP. I am currently working on this issue. I have already made changes. 😃